src/main/java/org/sonarsource/auth/github/GitHubIdentityProvider.java

```diff
@@ -153,4 +176,14 @@ private static String getName(GsonUser gsonUser) {
   private String generateUniqueLogin(GsonUser gsonUser) {
     return getKey() + "@" + gsonUser.getLogin();
   }
+
+  private enum TeamToGroup implements Function<GsonTeams.GsonTeam, String> {
+    INSTANCE;
+
+    @Override
+    public String apply(@Nonnull GsonTeams.GsonTeam gsonTeam) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this "Nonnull" annotation to honor the overridden method's contract. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2638)